tests/integration/cmd_dev_test.go

```diff
@@ -386,6 +386,47 @@ var _ = Describe("odo dev command tests", func() {
 				})
 			})
 		})
+
+		When("a delay is necessary for the component to start and running odo dev", func() {
+
+			var devSession helper.DevSession
+			var ports map[string]string
+
+			BeforeEach(func() {
+				helper.ReplaceString(filepath.Join(commonVar.Context, "devfile.yaml"), "npm start", "sleep 20 ; npm start")
```

Comment:
Would 10s suffice? Asking because this could set a precedent and subsequent tests like this would usually reuse `sleep 20`, thus slowing down the tests.

Reply:
I tried with 15s and it failed in IBM cloud tests from time to time.